### PR TITLE
bigger notification for black kois lobotomy

### PR DESCRIPTION
**Pull request by Sniblet**
Instead of two normal-sized lines using red and white font, which you're already getting a lot of in this stage of your lii'draic journey, the final, mind-shattering stage of black k'ois mycosis now gives you three lines of flavor text, each in fonts you're not seeing already: SPAN_GOOD, SPAN_HIGHDANGER, and then SPAN_NOTICE.
This is what you normally get:
image
This is what's replacing it:
image

### CODE REVIEW

File: code/modules/organs/subtypes/parasite.dm [hunk 1]
```diff
@@ -175,8 +175,9 @@
 		var/obj/item/organ/internal/brain/B = owner.internal_organs_by_name[BP_BRAIN]
 
 		if(B && !B.prepared)
-			to_chat(owner, "<span class='danger'>As the K'ois consumes your mind, you feel your past self, your memories, your very being slip away... only slavery to the swarm remains...</span>")
-			to_chat(owner, "<b>You have been lobotomized by K'ois infection. All of your previous memories up until this point are gone, and all of your ambitions are nothing. You live for only one purpose; to serve the Lii'dra hive.</b>")
+			to_chat(owner, "<span class='good'>As the K'ois consumes your mind, you feel your past self, your memories, your very being slip away... only slavery to the swarm remains...</span>")
```
Comment (Generalcamo) [inline code/modules/organs/subtypes/parasite.dm:178]:
```suggestion
			to_chat(owner, SPAN_GOOD("As the K'ois consumes your mind, you feel your past self, your memories, your very being slip away... only slavery to the swarm remains..."))
```

File: code/modules/organs/subtypes/parasite.dm [hunk 2]
```diff
@@ -175,8 +175,9 @@
...
+			to_chat(owner, "<span class='highdanger'>You have been lobotomized by K'ois infection.</span>")
```
Comment (Generalcamo) [inline code/modules/organs/subtypes/parasite.dm:179]: Add `SPAN_HIGHDANGER` to `_macros.dm`

File: code/modules/organs/subtypes/parasite.dm [hunk 3]
```diff
@@ -175,8 +175,9 @@
...
+			to_chat(owner, "<span class='notice'><b>All of your previous memories up until this point are gone, and all of your ambitions are nothing. You live for only one purpose; to serve the Lii'dra hive.</b></span>")
```
Comment (Generalcamo) [inline code/modules/organs/subtypes/parasite.dm:180]:
```suggestion
			to_chat(owner, SPAN_NOTICE("All of your previous memories up until this point are gone, and all of your ambitions are nothing. You live for only one purpose; to serve the Lii'dra hive."))
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

**Comment by Sniblet** [comment]
updated screenshots to actually compare, and it now complies
!review

**Comment by desvenlafaxine** [comment]
I would use different wording. Perhaps “Your brain has been consumed by...” Lobotomy just sounds odd.

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**